jobs/jobsv1/manager.go

```diff
@@ -123,9 +123,14 @@ func (s *manager) ExecuteWithinJob(ctx context.Context, accountID identity.DID,
 			mJob = tempJob
 		}
 
-		done <- true
+		// non blocking send
+		select {
+		case done <- true:
+		default:
+			log.Debug("job done channel has no listener")
+		}
 
-		if mJob != nil {
+		if mJob != nil && jobs.JobIDEqual(existingJobID, jobs.NilJobID()) {
```

Comment:
Just sent on the parent job?

Reply:
yup, as child jobs are irrelevant to the users, as well as can explode the number of webhooks needlessly.